confuse.py

```diff
@@ -778,19 +778,37 @@ def check_plain(self):
         plain = super(Loader, self).check_plain()
         return plain or self.peek() == '%'
 
+    @staticmethod
+    def override(loader):
+        """Overrides yaml Loader constructors with Confuse fancy
+        constructors. Call this method on your custom
+        Loader before Configuration intialization and if you need
+        enhanced Confuse's lists and ordered dicts
+        """
```

Comment:
Docstrings usually use the imperative mood. How about this as a simple way to put it?

> Modify a PyYAML `Loader` class to add extra constructors for strings and maps. Call this method on a custom Loader class to make it behave like Confuse's own `Loader`.